inst/report/KRIReportByQTL.Rmd

```diff
@@ -8,7 +8,7 @@ output:
        smooth_scroll: true
     css: styles.css
 params:
-    assessment: NA
+    assessment: NULL
```

Comment:
This is `NA` in the Country report - not sure if it matters but we should consider making them all consistent in a future PR.